.xo-config.json

```diff
@@ -11,6 +11,12 @@
 		"__filebasename"
 	],
 	"rules": {
+		"@typescript-eslint/no-unsafe-assignment": "warn",
+		"@typescript-eslint/no-unsafe-member-access": "warn",
+		"@typescript-eslint/no-unsafe-return": "warn",
+		"@typescript-eslint/no-unsafe-call": "warn",
```

Comment:
We have `any` in a lot of places, especially but not only due to API calls. I don't know if we can fix these without making API processing verbose. We specifically use `any` in there to avoid having to type the API. Any code we add won't make the code any safer